.github/actions/set-env-variables/action.yml

```diff
@@ -16,7 +16,22 @@ runs:
         echo "CILIUM_CLI_VERSION=$CILIUM_CLI_VERSION" >> $GITHUB_ENV
         echo "CILIUM_CLI_IMAGE_REPO=quay.io/cilium/cilium-cli-ci" >> $GITHUB_ENV
         echo "CILIUM_CLI_SKIP_BUILD=true" >> $GITHUB_ENV
-        echo "CILIUM_CLI_CODE_OWNERS_PATHS=CODEOWNERS" >> $GITHUB_ENV
+
+        # Detect if we're on a stable branch (vX.Y pattern) and include TESTOWNERS if available
+        # To create TESTOWNERS on a stable branch: cp CODEOWNERS TESTOWNERS && git add TESTOWNERS && git commit
+        BRANCH_NAME="${GITHUB_REF_NAME:-$(git rev-parse --abbrev-ref HEAD)}"
```

Comment:
I'm not entirely sure if `GITHUB_REF_NAME` produces the expected result in all of the different ways we trigger GitHub actions, we'll want to double-check this (or go with an option that doesn't rely on branch names).